tests/PersistentTest.php

```diff
@@ -378,3 +394,19 @@ private function noopGlobalFunction(): PersistentFunctionMock {
 		return Persistent\noop_global_function('Hammock\Fixtures\return_input');
 	}
 }
+
+class MockPersistentMockRegistry extends Persistent\PersistentMockRegistry {
+	public static function getRegistry(): vec<Interfaces\IDeactivatable> {
+		return self::$registry;
+	}
+}
+
+class MockDeactivatable implements Interfaces\IDeactivatable {
```

Comment:
Would you be able to group these test classes under `tests/Fixtures/` along with the rest of the test classes?

Reply:
Yeah definitely!